wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -219,7 +222,7 @@ def _check_executable_mismatch(
         token: tokenize.TokenInfo,
         *,
         is_shebang: bool,
-    ) -> None:
+    ) -> None:  # pragma: not-win
```

Comment:
Can we move it to `if is_windows() or self.filename == STDIN:` line?

Reply:
I think, we can not because testing shebang violations in this request  was disabled.